php/lang/security/injection/tainted-exec.yaml

```diff
@@ -0,0 +1,51 @@
+rules:
+- id: tainted-exec
+  languages:
+  - php
+  severity: WARNING
+  message: <- 
```

Comment:
```suggestion
  message: >-
```

We usually go for this operator. It might change how multiline strings are displayed in the CLI. I'm not sure anymore what the exact reason is. I think there's a CI check for this as well.

Reply:
This recommendation is included in the docs as well:
https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository#rule-messages